sql/src/main/java/io/crate/execution/dsl/projection/WindowAggProjection.java

```diff
@@ -118,6 +139,8 @@ public ProjectionType projectionType() {
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         windowDefinition.writeTo(out);
+        out.writeGenericValue(startFrameOffset);
```

Comment:
I think we'll need to add version handling in the streaming here. Since we're going to support rolling upgrades from 4.0 to 4.1 we must not send those values to a node running 4.0 and on the other hand, must not attempt to read these values if the node sending them is running 4.0